index.js

```diff
@@ -176,47 +186,34 @@ Client.ready().then(Connection => {
           
           return _Tx
         })
-        
-        // console.dir(Transactions[0], { depth: null })
-        // process.exit(1)
```

Comment:
i had kept these commented blocks in previously, but at this point they would not be correct